### PR TITLE
Browserify

### PR DESCRIPTION
**Pull request by calvinmetcalf**
its bigger because I'm including the bundled files, haven't ported the tests because I haven't gotten around to untangling this flight jasmin stuff.

this addresses none of the issues brought up in #48 but one idea might be to use browserify as a test, if you can install it, then you can much around inside and adapt stuff, if you can't then you have no business mucking around inside and you can only change the config.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,6 +77,6 @@ <h5 data-project="contact">Please send feedback, ideas, and bug reports to our <
         </div>
       </div>
     </div>
-
+    <script type="text/javascript" src="dist/bundle.js"></script>
```
Comment (paulswartz) [inline index.html]: This is what I want to avoid.  Now you have to make a build for Finda to work, instead of just editing the files.

**Comment by paulswartz** [comment]
> if you can install it, then you can much around inside and adapt stuff, if you can't then you have no business mucking around inside and you can only change the config.

The idea of #48 is that there's a build for people who have no interest in development, not that we deliberately make developing Finda harder for people.

**Comment by calvinmetcalf** [comment]
Yeah your probably right and going through this it made me think this app
isn't very big and we almost spend more time configing it then writing it,
we might be better off ditching modules all together. AMD and browserify
both have humps that are probably not worth it for something this small.

**Comment by calvinmetcalf** [comment]
ok @paulswartz round 2
- able to create a minified bundle
- don't break anything else
- had to put the config into the html as opposed to the file so we could reuse it with browserify
- switched it over to consistently use `define(function(require, exports, module) {` syntax

**Comment by paulswartz** [comment]
So I took a quick stab at this with Grunt, keeping the RequireJS workflow around.  It's pretty straightforward, and I ended up with #90.

**Comment by calvinmetcalf** [comment]
you could probably do this without grunt and it would be approximately 4x easier

**Comment by calvinmetcalf** [comment]
also I'm out in the other room if your at code for boston

File: index.html [hunk 2]
```diff
@@ -50,7 +50,35 @@
         <button type="button" class="close">&times;</button>
       </div>
     </div>
-
+    <script>
+    var require = {
```
Comment (paulswartz) [inline index.html]: Do you still need this, since it's included in `src/script.js`?

**Pull request merged**